rpm/SPECS/openresty.spec

```diff
@@ -164,10 +160,6 @@ mkdir -p %{buildroot}/etc/init.d
 export QA_RPATHS=$[ 0x0002 ]
 
 
-%clean
-rm -rf %{buildroot}
-
```

Comment:
@zxcvbn4038 Since this is automatically generated by the `rpmspec`, I guess it might be required for older versions of rpmbuild. Not sure. I suggest we preserve it (and it is also not related to the current PR's topic).